libs/dh/admin/feature-user-management/src/lib/tabs/users-tab/dh-users-tab.component.ts

```diff
@@ -95,12 +107,25 @@ export class DhUsersTabComponent {
   readonly pageIndex$ = this.store.paginatorPageIndex$;
   readonly pageSize$ = this.store.pageSize$;
 
-  readonly isLoading$ = this.store.isLoading$;
+  readonly isLoading$ =
+    this.store.isLoading$ ||
```

Comment:
I would not expect this to work. You are just testing for i isLoading$ variable is defined or not.